### ISSUE TITLE
Generic usage is wrong in model specific classes

### ISSUE THREAD

**Issue by maxleiko** [kevoree-modeling/framework#8]
Generated source for model-specific classes **XXXModel**, **XXXUniverse** uses generic poorly as they all use `any` in their signature:

``` ts
class KevoreeModel extends org.kevoree.modeling.abs.AbstractKModel<any> {
}
```

instead of 

``` ts
class KevoreeModel extends org.kevoree.modeling.abs.AbstractKModel<KevoreeUniverse> {
}
```

and

``` ts
class KevoreeUniverse extends org.kevoree.modeling.abs.AbstractKUniverse<any, any> {
}
```

instead of 

``` ts
class KevoreeUniverse extends org.kevoree.modeling.abs.AbstractKUniverse<KevoreeView, KevoreeUniverse> {
}
```

nb: that might **not** be isolated cases, I did not check all the generated code

**Comment by dukeboard** [comment]
Problem is isolated in J2TS, will be enhanced soon

**Comment by dukeboard** [comment]
It should be fixed in J2TS, greg can you look at it ? (i already checked and at the moment we fill by any in the typescript , that's explains the problem, a simple better print should fix the issue)

**Comment by gnain** [comment]
Just committed a potential fix: kevoree-modeling/java2typescript@ec3fc49d4a2f5e98f2c61224c4b58407ce1de1d7
@maxleiko Could you try with j2TS 45-SNAP ?

**Comment by maxleiko** [comment]
Ok, looks like **XXXModel, XXXUniverse and XXXView** are properly using generics with j2ts 46-SNAPSHOT.  
I did not check elsewhere though
